scripts/destroy_infrastructure.sh

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env bash
 
-set -e
+set -euxo pipefail
 
 TERRAFORM_STATE_BUCKET="${TERRAFORM_STATE_BUCKET:-eq-terraform-load-generator-tfstate}"
 
-terraform init --upgrade --backend-config prefix=${PROJECT_NAME} -var "project_name=${PROJECT_NAME}" --backend-config bucket=${TERRAFORM_STATE_BUCKET}
+tfenv use "$(< .terraform-version)"
```

Comment:
tfenv shouldn't be part of this?. I may have installed terraform without using tfenv?

Reply:
Discussed this with @ajmaddaford - we should move towards not running tf locally at all and use the tasks via fly execute - in that case we know that the script will be used via the build image and have tfenv.

Reply:
We should look at that after this merges. For now it keeps the concourse yaml clean 🙂